osquery/tables/events/linux/process_events.cpp

```diff
@@ -158,7 +166,7 @@ Status AuditProcessEventSubscriber::ProcessEvents(
         row["owner_gid"], first_path_event_record->fields, "ogid", "0");
 
     // Parent is currently not supported on Linux.
-    row["parent"] = "-1";
+    row["parent"] = row["ppid"];
```

Comment:
Can you move this above, right after fetching the information so that this becomes obvious? Also I would prefer passing a temp variable to the getter function as oppose to add it to the map.